content/Watsonx/WatsonxAI/100.md

```diff
@@ -13,7 +13,7 @@ toc: true
 ### [watsonx Assistant and Watson Discovery](https://techzone.ibm.com/collection/watson-enterprise/environments)
 
 
-## FAQs
+# FAQs
```

Comment:
suggestion: if you want the headings to show up in the TOC (table of contents) not he right hand then make this line an h2 _(or remove it)_ and also make the following line an h2...
https://github.com/ibm-build-lab/VAD-VAR-Workshop/blob/7ada6ea923977affb0eb3483f2f73638791290e0/content/Watsonx/WatsonxAI/100.md?plain=1#L20

Reply:
I went ahead and removed it.  Good catch!